test/helper.go

```diff
@@ -139,6 +139,8 @@ func NewHelper(t *testing.T) *Helper {
 		// Set the healthcheck interval to 1 second for testing
 		helper.Env().Config.HealthCheck.HeartbeartInterval = 1
 		helper.HealthCheckServer = server.NewHealthCheckServer()
+		// Disable TLS for testing
+		helper.Env().Config.GRPCServer.DisableTLS = true
```

Comment:
why do you disable TLS here? I saw you are using OCP serving certs to generate certs.

Reply:
tls is disabled for integration testing, this is how `helper.Env().Config.GRPCServer.DisableTLS = true` works.
but for e2e testing, it is enabled, we generate the openshift serving certificate for grpc broker TLS.